include/posix/unistd.h

```diff
@@ -11,11 +11,43 @@ extern "C" {
 #endif
 
 #include "sys/types.h"
+#include "sys/stat.h"
+
+#ifdef CONFIG_POSIX_FS
+#include <fs.h>
+
+typedef struct fs_dir_t DIR;
+typedef unsigned int mode_t;
+
+struct dirent {
+	unsigned int d_ino;
+	char d_name[MAX_FILE_NAME + 1];
```

Comment:
Might want to define PATH_MAX instead in limits.h, that is the standard value.  This minor, though, so please create an issue to track this.